Modules/posixmodule.c

```diff
@@ -7957,13 +7957,13 @@ static PyObject *
 os_getpid_impl(PyObject *module)
 /*[clinic end generated code: output=9ea6fdac01ed2b3c input=5a9a00f0ab68aa00]*/
 {
-#ifdef MS_WINDOWS
+#ifdef MS_WINDOWS_UWP
     return PyLong_FromPid((pid_t)GetCurrentProcessId());
```

Comment:
Should probably just take this `FromUnsignedLong`, in case the cast turns out to be problems (POSIX types are often signed integers where Windows uses unsigned).